mtp_api/apps/prison/forms.py

```diff
@@ -1,6 +1,82 @@
+import csv
+import decimal
+import io
+
 from django import forms
+from django.contrib.admin.widgets import AdminFileWidget
+from django.db import transaction
+from django.forms.models import ModelChoiceField
 from django.utils.translation import gettext_lazy as _
 
+from prison.models import Prison, PrisonerBalance, validate_prisoner_number
+
 
 class LoadOffendersForm(forms.Form):
     modified_only = forms.BooleanField(label=_('Only load updated offender locations'), required=False)
+
+
+class PrisonerBalanceUploadForm(forms.Form):
+    csv_file = forms.FileField(label=_('Prisoner balances file'), allow_empty_file=True, widget=AdminFileWidget)
+    prison = ModelChoiceField(queryset=Prison.objects.all().order_by('-private_estate', 'nomis_id'))
```

Comment:
Now we have the `use_nomis_for_balances` field (added in a subsequent commit to this addition) - can we filter by that for the ChoiceField, (reducing room for error in selecting Birmingham essentially!)

Reply:
yeah good idea @mattbee